### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GTConnor**
Add empty line

### CODE REVIEW

**Comment by GTRekter** [comment]
@GTConnor can you translate the content from Italian to English?

**Comment by GTConnor** [comment]
> @GTConnor can you translate the content from Italian to English?

Will do

**Review by GTRekter — COMMENTED** [review]
@GTConnor can you translate the content from Italian to English?

File: README.md [hunk 1]
```diff
@@ -12,3 +12,4 @@ Please feel free to contribute by adding new templates to this project:
 - Create a new Pull Request
 
 ## Code of Conduct
```
Comment (GTRekter) [inline README.md:14]: @GTConnor can you translate the content from Italian to English?

Comment (GTConnor) [reply README.md:14]: I will have an update by tomorrow

**Pull request merged**